docs/examples/travel_agent/tools/slack.py

```
@@ -0,0 +1,131 @@
+        f"✈️ *Travel Approval Request*\n\n"
+        f"*Employee:* {employee_name}\n"
+        f"*Route:* {origin} → {destination}\n"
+        f"*Dates:* {departure_date} – {return_date}\n"
+        f"*Flight:* {flight_summary}\n"
+        f"*Hotel:* {hotel_summary}\n"
+        f"*Estimated Total:* ${total_cost:,.2f}\n\n"
+        f"Please reply ✅ to approve or ❌ to reject."
+    )
+
+    # --- Real Slack send ---
+    if _SLACK_SDK_AVAILABLE and token and token != "xoxb-your-slack-bot-token-here" and manager_id:
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "token"

(S105)